### PR TITLE
Microsoft.SurfaceDuoEmulator-Android11 version 2021.824.90

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28510)

### CODE REVIEW

File: manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.SurfaceDuoEmulator-Android11
```
Comment (jedieaston) [inline manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.SurfaceDuoEmulator.Android11
```

File: manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.SurfaceDuoEmulator-Android11
```
Comment (jedieaston) [inline manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.SurfaceDuoEmulator.Android11
```

File: manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.SurfaceDuoEmulator-Android11
```
Comment (jedieaston) [inline manifests/m/Microsoft/SurfaceDuoEmulator-Android11/2021.824.90/Microsoft.SurfaceDuoEmulator-Android11.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.SurfaceDuoEmulator.Android11
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Just to keep up with the convention of separating incompatible manifests with a "." instead of a "-". If someone else is fine with it then they can approve it 😊

(don't forget to move it to the right folder too)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**